doc/release-notes-gui-459.md

```diff
@@ -0,0 +1,4 @@
+GUI changes
+-----------
+
+- The Bech32 checkbox has been replaced with a dropdown for all address types, including the new Bech32m (BIP-350) standard for Taproot enabled wallets. The default remains Bech32 (SegWit).
```

Comment:
```suggestion
- The Bech32 checkbox has been replaced with a dropdown for all address types, including the new Bech32m (BIP-350) standard for Taproot enabled wallets.
```

nit: I don't think we need to mention stuff that doesn't change. Also, this is wrong? The default is whatever `-addresstype` is set to?

Reply:
Fixed